data/ui_dashboards/molgenis.csv

```diff
@@ -0,0 +1,69 @@
+Charts,,,y axis min value,int,,,,,,,"http://semanticscience.org/resource/SIO_000453,http://semanticscience.org/resource/SIO_001113",A value that indicates the minimum threshold to display on the y-axis (applies only to continuous variables)
+Charts,,,y axis max value,int,,,,,,,"http://semanticscience.org/resource/SIO_000453,http://semanticscience.org/resource/SIO_001114",A value that indicates the minimum threshold to display on the y-axis (applies only to continuous variables)
+Charts,,,y axis ticks,int_array,,,,,,,"http://semanticscience.org/resource/SIO_000453,http://semanticscience.org/resource/SIO_000915,http://semanticscience.org/resource/SIO_001316",An array of numbers that indicate the breakpoints on the y-axis
+Charts,,,top margin,int,,,,,15,"(function () {
+  if (topMargin !== null && topMargin < 0) {
+  return ""The top margin cannot be less than 0""
+  }
+  if (topMargin !== null && topMargin > 250) {
+  return ""The top margin cannot be more than 250""
+  }
+})();",http://purl.obolibrary.org/obo/NCIT_C69210,A number that indicates the amount of white space to add to the top of the chart
+Charts,,,right margin,int,,,,,15,"(function () {
+  if (rightMargin !== null && rightMargin < 0) {
+  return ""The right margin cannot be less than 0""
+  }
+  if (rightMargin !== null && rightMargin > 250) {
+  return ""The right margin cannot be more than 250""
+  }
+})();",http://purl.obolibrary.org/obo/NCIT_C69201,A number that indicates the amount of white space to add to the right of the chart
+Charts,,,bottom margin,int,,,,,60,"(function () {
+  if (bottomMargin !== null && bottomMargin < 0) {
+  return ""The bottom margin cannot be less than 0""
+  }
+  if (bottomMargin !== null && bottomMargin > 250) {
+  return ""The bottom margin cannot be more than 250""
+  }
+})();",http://purl.obolibrary.org/obo/NCIT_C69210,A number that indicates the amount of white space to add to the bottom of the chart
+Charts,,,left margin,int,,,,,80,"(function () {
+  if (leftMargin !== null && leftMargin < 0) {
+  return ""The left margin cannot be less than 0""
+  }
+  if (leftMargin !== null && leftMargin > 250) {
+  return ""The left margin cannot be more than 250""
+  }
+})();",http://purl.obolibrary.org/obo/NCIT_C69201,A number that indicates the amount of white space to add to the left of the chart
+Charts,,,legend position,ontology,,,Legend positions,,top,,"http://semanticscience.org/resource/SIO_000468,http://semanticscience.org/resource/SIO_000056",The position of the legend around the chart.
+Charts,,,data points,refback,,,Chart data,included in chart,,,http://semanticscience.org/resource/SIO_000546,Data that is prepared to be used in a chart
+Charts,,,dashboard page,ref,,,Dashboard pages,,,,http://semanticscience.org/resource/SIO_000302,
+Chart data,,,,,,,,,,,http://semanticscience.org/resource/SIO_000089,Data used in a visualisation
```

Comment:
it is a bit risky, but you could consider to have the data in a big json blob instead?

Reply:
My intention for this schema is to allow data managers (and other users) to control the data that is displayed in a chart without having to update the vue application. For example, if a user wanted to regroup data, they can do so in this table by editing the rows or importing the updated dataset. I was tempted to have an option for other data structures, but users wouldn't be able to modify the chart data as easily. This model may also serve as the starting point for the chart builder.